pyproject.toml

```diff
@@ -1,3 +1,39 @@
+[project]
+name = "mimir-worker-k8s"
+version = "0.1"  # this is in fact irrelevant
```

Comment:
Can/should we omit this section, or use "0.0"?

Reply:
We cannot emit this because:
```
ValueError: invalid pyproject.toml config: `project`.
configuration error: `project` must contain ['version'] properties
```